TypeCobol/Tools/CommandLine/TypeCobolCompiler.cs

```diff
@@ -23,7 +23,7 @@ public static void Main(string[] args)
 
             string sourcePath = projectRootPath + @"TypeCobol.Test\Samples\EI Cobol samples\EI-Production";
             string[] programExtensions = { ".PGM" };
-            string[] copyExtensions = { ".CPY" };
+            string[] copyExtensions = { ".CPY", ".COPY" };
```

Comment:
Can be replaced by Helpers.DEFAULT_COPY_EXTENSIONS

Reply:
done.